src/Cmd/Remove.cc

```diff
@@ -0,0 +1,57 @@
+#include "Remove.hpp"
+
+#include "../Cli.hpp"
+#include "../Logger.hpp"
+#include "../Manifest.hpp"
+#include "../Rustify/Result.hpp"
+
+#include <cstdlib>
+#include <fmt/ranges.h>
+#include <fstream>
+#include <string>
+#include <toml.hpp>
+#include <toml11/types.hpp>
+#include <vector>
+
+namespace cabin {
+
+static Result<void> removeMain(CliArgsView args);
+
+const Subcmd REMOVE_CMD = Subcmd{ "remove" }
```

Comment:
For consistent format:

```suggestion
const Subcmd REMOVE_CMD =  \\
    Subcmd{ "remove" }
```